crates/tessera-backbone/src/application/path/mod.rs

```diff
@@ -52,6 +60,12 @@ impl From<Path> for PathData {
 
 #[derive(thiserror::Error, Debug)]
 pub(crate) enum Error {
+    #[error("Entered Path({entered_path}) is already registered")]
+    PathDuplicated { entered_path: String },
+    #[error("parent path for Path({entered_path}) is not registered")]
```

Comment:
nit: p -> P (if there's a error message rule that should be uppercase at the beginning)